### ISSUE TITLE
NA introduced into position when opening curly braces

### ISSUE THREAD

**Issue by andyquinterom** [REditorSupport/languageserver#564]
The language server introduces NA when code like the following occurs

```R
function() {
  test <- print(
}
```

This causes for the Neovim lsp interface to error out. In this like shiny this is a very common thing to find. 

My fix was pretty easy, I went to the `code_point_to_unit` function and added the following:

```R
result[is.na(pts)] <- m
```

This fixes the issue and does not seem to break any other behaviour. This is a pretty minor fix. Hope it can get merged!

**Comment by andyquinterom** [comment]
Some more context:

This is the error I was getting on Neovim after configuring the LSP.
![image](https://user-images.githubusercontent.com/51521150/180841472-7e70ebd0-6627-4bf8-9d8a-e3317366b8b4.png)

I modified the neovim source code to expose the exact comparison that was causing the error. See that is tries to compare a string with a number.

After exposing the variable is was a string containing `NA`. This is pretty common R stuff to from there I tried to make my fix.
